web/client/epics/usersession.js

```diff
@@ -96,21 +97,24 @@ export const saveUserSessionEpicCreator = (nameSelector, sessionSelector, idSele
  * @param {*} startAction the action type that will start saving of the user sessions
  * @param {*} endAction the action type that will stop saving of the user sessions
  * @param {*} frequency interval between saves (in milliseconds)
+ * @param {*} finalAction optional action emitted after stop
```

Comment:
finalAction looks a function while the other seems to be objects. Maybe we should make the types uniform (and indicate object/function in jsdoc accordingly).

Reply:
agree for jsdoc, but startAction and endAction are action types (strings) while finalAction is an action creator function